src/serverMain/kotlin/com/studo/campusqr/Server.kt

```diff
@@ -105,6 +105,7 @@ suspend fun main() {
 
         route("{id}") {
           post("visit") { call.visitLocation() }
+          post("checkOut") { call.checkOutLocation() }
```

Comment:
I think all paths should be lowercase. We have not just this but many camelCase paths but I think that it's a convention to use lowercase uri paths.

Reply:
yeah that's true

Reply:
fixed in https://github.com/studo-app/campus-qr/pull/36/commits/2968343870a2d247118ce42369b7031f9d842a94